src/js/components/TaskTable.js

```diff
@@ -50,6 +52,15 @@ class TaskTable extends React.Component {
         sortFunction
       },
       {
+        className: this.getHostColumnClassname,
```

Comment:
When using `cacheCell: true`, the cell doesn't update properly.

Using caching:
![](https://s3.amazonaws.com/f.cl.ly/items/1A07060d3i2t0g1G472k/Screen%20Shot%202016-07-15%20at%2010.18.11%20AM.png?v=81492015)

Not using caching:
![](https://cl.ly/0X44241a0m06/Screen%20Shot%202016-07-15%20at%2010.18.34%20AM.png)